### ISSUE TITLE
Create experimental setup, determine parameters ranges and sampling

### ISSUE THREAD

**Issue by EwoutH** [EwoutH/urban-self-driving-effects#7]
### Variables to vary

- self-driving VoT
- self-driving price
- self-driving car density
- Expected induced demand (gained from percieved travel time costs reductions from earlier experiments)

(policies if feasible)

### Parameters ranges
- self-driving VoT: 1.0 to 0.25, compared to a car.
- self-driving price: 1.5 to 0.1, with some proper sampling to get goo density at the low end
- self-driving car density: 1.5 (meaning less dense) to 0.5 (meaning twice as dense)

### Sampling

Probably latin hypercube. But needs research.

**Comment by EwoutH** [comment]
- Seed analysis
- Correlation between self-driving VoT and car density
- 3x3x3x3 matrix-ish (full factorial)

Consider (adaptive) sparse grid

KPIs: Travel time reduction, cost reduction, etc.

**Comment by EwoutH** [comment]
@quaquel I'm trying a ~100 sample run overnight, to see if there are any problems with my data pipelines etc.

I was thinking of sampling with equal ratios, so like this:

![image](https://github.com/user-attachments/assets/fcd3bfde-58f3-4d19-b308-a8e72e3a43aa)
linear x-scale

![image](https://github.com/user-attachments/assets/866463a9-e734-4dbf-af7d-0a59b0508d83)
log x-scale

And then in a Latin Hypercube.

The motivation behind is because I expect the difference between for example 0.1 and 0.2 AV costs to be huge, so I want higher density there. So equal ratios seems to make sense.

What do you think?

**Comment by EwoutH** [comment]
@quaquel do you have a view on linear vs equal-ratio sampling?

**Comment by quaquel** [comment]
yes, but not one that is easily shared in a few sentences. I prefer uniform sampling because it gives you insight into the behavior across the space. At least here you have uniform sampling in log space so that helps.

**Comment by EwoutH** [comment]
If your advice is "don't do it" that's als perfectly fine!

Let's discuss Wednesday in more detail.

<hr>

In other news, thanks to the amazing people at [MRDH](https://mrdh.nl/verkeersmodel), parking pressure as a metric is back on the map!

![inbound_outbound_traffic](https://github.com/user-attachments/assets/42338144-75cc-48bb-81ad-5650a79711c1)

The bottom graph is most interesting, you're seeing in some regions almost 80% of the traffic being inbound (red) or outbound (blue). 

1 is Rotterdam Centrum, 16-19 is Vondelingenplaat, Botlek, Europoort, Maasvlakte, with low traffic in general but (almost) no residents, so all traffic is work-related.


And of course this will also help with making the whole model more realistic, including (probably) more traffic jams in the morning rush hours and somewhat less in the afternoon.

**Comment by quaquel** [comment]
> If your advice is "don't do it" that's als perfectly fine!

Let's indeed discuss

Nice figures!

**Comment by EwoutH** [comment]
I started a full-factorial test run overnight with the following parameter combinations:
```Python
av_cost_factor = [1.0, 0.5, 0.25]
av_vot_factor = [1.0, 0.5, 0.25]
av_density = [1.5, 1.0, 0.5]
induced_demand = [1.0, 1.25, 1.50]
```
Unfortunately, it failed after 13 of the 81 runs, due to a very rare bug (hopefully fixed in 4d63a47), but it saved the results from the runs finished, so we can start with that.

However, we now need to go from time-series, multi-area and distribution (mode choice) metrics to single-value metrics, to allow visualizing them properly. The initial idea is to generate some dimensional-stacked heatmaps, like:

![image](https://github.com/user-attachments/assets/db78bd0a-9d53-46e8-83e9-4cb29e6db888)

The following variables will be visualized:
- [x] number of journeys (only impacted by induced demand)
- [x] av mode share
- [x] car mode share
- [x] mean travel time
- [x] mean travel speed
- [x] comfort and perceived comfort costs
- [x] total travel volume (summed over areas, averaged over time)
- [x] total travel time (summed over areas, averaged over time)
- [x] 95% vehicles remain (summed over areas, 95% high over time)
- [x] average delay (weighted by area volume over time)
- [x] average speed (weighted by vehicles remain over time)
- [ ] % peak increased parked cars in area 1 (Rotterdam Centrum)
- [x] Save perceived costs

If any measures are correlating highly, one can be dropped in favor of the other.

The most extreme / interesting runs can be examined in more detail.

**Comment by EwoutH** [comment]
Potential policies
- Reduce the speed limit on 0% / 25% / 50% of all roads  (network level)
- Add tax €5 / €10 or don't allow at all if area is above certain congestion threshold (adaptive policy)
